lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb

```diff
@@ -8,25 +8,32 @@ class XcodebuildCommand
       "watchsimulator" => "watchOS"
     }.freeze
 
+    DESTINATION_OF_SDK = {
+      "iphoneos" => "\"generic/platform=iOS\"",
+      "iphonesimulator" => "\"generic/platform=iOS Simulator\""
+    }.freeze
+
     def self.xcodebuild(options)
       sdk = options[:sdk] || "iphonesimulator"
       targets = options[:targets] || [options[:target]]
-      platform = PLATFORM_OF_SDK[sdk]
+      # platform = PLATFORM_OF_SDK[sdk]
 
       cmd = ["xcodebuild"]
       cmd << "-project" << options[:sandbox].project_path.realdirpath.shellescape
       targets.each { |target| cmd << "-target" << target }
       cmd << "-configuration" << options[:configuration]
       cmd << "-sdk" << sdk
-      unless platform.nil?
-        cmd << Fourflusher::SimControl.new.destination(:oldest, platform, options[:deployment_target])
-      end
+      # unless platform.nil?
+      #   cmd << Fourflusher::SimControl.new.destination(:oldest, platform, options[:deployment_target])
+      # end
+      cmd << "-destination" << DESTINATION_OF_SDK[sdk] if DESTINATION_OF_SDK.key?(sdk)
```

Comment:
For now, can we fall back to the previous logic with `Fourflusher` in case the `sdk` is not present in `DESTINATION_OF_SDK`?

I'm not quite sure if this plugin is currently used for other platforms (ie. watch os, tv os...) or not. I haven't well tested the logic for those platforms.